api/helpers/user.js

```diff
@@ -5,10 +5,14 @@ let unless = require("express-unless");
 const getUserIfExists = async req => {
   let user = req.user;
   if (user) {
-    user = await db.one(sql("../sql/user_by_email.sql"), {
+    user = await db.oneOrNone(sql("../sql/user_by_email.sql"), {
```

Comment:
Weird, I thought we'd made this change awhile ago. Looks reasonable in any case.